### ISSUE TITLE
Incorrect CSP directive used when repeated

### ISSUE THREAD

**Issue by Rob--W** [mozilla/addons-linter#4575]
### Describe the problem and steps to reproduce it:

The implementation of the MANIFEST_CSP and MANIFEST_CSP_UNSAFE_EVAL rules rely on a CSP validator.
A CSP source list may potentially contain multiple sources, and in that case the first one should be used, and any duplicates be ignored.

Test case, create `manifest.json` and lint the directory containing the following:
```json
{
  "name": "x",
  "version": "1",
  "manifest_version": 2,
  "content_security_policy": "script-src 'unsafe-eval'; script-src 'self'"
}
```

### What happened?

The addons-linter currently uses the last one instead of the first one. No errors are emitted.

### What did you expect to happen?
The addons-linter should use the first one for validation, and emit a warning for the presence of 'unsafe-eval'.

### Anything else we should know?

I have a patch up at #4573

**Comment by AlexandraMoga** [comment]
I've verified this issue with the latest linter release (5.22.0) and CSP warnings are now raised for both MANIFEST_CSP and MANIFEST_CSP_UNSAFE_EVAL.

[before fix]
![image](https://user-images.githubusercontent.com/31961530/202166585-2c520a22-659f-4b85-969c-182194bb8c0d.png)


[after fix]
![image](https://user-images.githubusercontent.com/31961530/202166667-ff49ebf4-1a22-4c64-bdb1-c84c321598eb.png)
